docs/integration/ibm-cloud-kubernetes-logs.md

```diff
@@ -0,0 +1,253 @@
+
+### 2. Add the `logagent.conf` as a `ConfigMap`
+
+Create the ConfigMap from the `logagent.conf` file. Run this command from the dir where you have the `logagent.conf`:
+
+```bash
+kubectl create configmap logagent-config --from-file=./logagent.conf
+```
+
+### 3. Create the Logagent DaemonSet
+
+Create the [ibm-cloud-logagent-with-config-ds.yml](https://github.com/sematext/logagent-js/blob/master/kubernetes/ibm-cloud-logagent-with-config-ds.yml) DaemonSet file.
+
+```bash
+curl -o ibm-cloud-logagent-with-config-ds.yml https://raw.githubusercontent.com/sematext/logagent-js/master/kubernetes/ibm-cloud-logagent-with-config-ds.yml 
+```
+
+You don't need to change anything as all the config is in `logagent.conf`, just run the DaemonSet:
+
+```bash
+kubectl create -f ibm-cloud-logagent-with-config-ds.yml
+```
+
+**When you want to edit the config, change the `logagent.conf`, recreate the ConfigMap, restart the Logagent pod to grab the new ConfigMap and you're done!**
+
+Continue reading below to see how to configure more advanced settings.
+
+### Drop Noisy Container Logs 
+
+Edit the `logagent.conf` to add the [drop-events](../logagent/output-filter-dropevents/) outputFilter.
+
+```yaml hl_lines="13 14 15 16 17 18 19"
+# logagent.conf
+options:
+  debug: false
+
+input: 
+  files: 
+    - /var/log/*.log
+    - /var/log/containers/*.log
+
+inputFilter:
+  - module: input-filter-k8s-containerd
+
+outputFilter:
+  dropEvents:
+    module: drop-events
+    filters:
+      message:
+        include: !!js/regexp /critical|auth|error|failed/
+        exclude: !!js/regexp /status/i
+
+output:
+  elasticsearch:
+    module: elasticsearch
+    url: https://logsene-receiver.sematext.com
+    index: YOUR_SEMATEXT_LOGS_TOKEN
+
+```
+
+In the `filters` section you can pick a field to apply regex mathing to.
+If there's a match you can either include or exclude that particular log line.
+The sample above will include all log lines that match `critical|auth|error|failed` but exclude all that match `status`.
+
+### Forward Container Logs to Multiple Apps with Log Routing
+
+To enable log routing you edit the output plugin to use multiple indices.
+Under the token value you need to add a regex for the log file you want to match.
+
+```yaml hl_lines="24 25 26 27 28"
+# logagent.conf
+options:
+  debug: true
+
+input: 
+  files: 
+    - /var/log/*.log
+    - /var/log/containers/*.log
+
+inputFilter:
+  - module: input-filter-k8s-containerd
+
+outputFilter:
+  dropEventsFilter:
+    module: drop-events
+    filters:
+      message:
+        exclude: !!js/regexp /status/i
+
+output:
+  elasticsearch:
+    module: elasticsearch
+    url: https://logsene-receiver.sematext.com
+    indices: 
+      b0e9f481-xxxx-xxxx-xxxx-3ff20227d3d3: # All logs except kube-system
+        - ^(?!.*(kube-system).*).*\.log
+      9365eb2f-xxxx-xxxx-xxxx-5a833072353f: # Only kube-system logs
+        - .*kube-system.*\.log
+
+```
+
```

Comment:
Maybe we want to say something like:
In place of b0e9f481-xxxx-xxxx-xxxx-3ff20227d3d3 and 9365eb2f-xxxx-xxxx-xxxx-5a833072353f in the example above you would use your own Sematext Apps logs tokens.

Reply:
Nice, added.